Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyTreeView.cxx

```diff
@@ -1504,6 +1506,23 @@ void qMRMLSubjectHierarchyTreeView::onSelectionChanged(const QItemSelection& sel
   emit currentItemsChanged(selectedShItems);
 }
 
+//------------------------------------------------------------------------------
+void qMRMLSubjectHierarchyTreeView::onCurrentSelection(const QModelIndex &currentItemIndex)
+{
+    Q_D(qMRMLSubjectHierarchyTreeView);
+    if (!d->SortFilterModel)
+      {
+      return;
+      }
+
+    vtkIdType itemID = d->SortFilterModel->subjectHierarchyItemFromIndex(currentItemIndex);
+    // Emit current item signal only if the current item is pressed
```

Comment:
This comment "translates" the below code to an English sentence. Comments are more useful to explain the why instead of the what (that we can see anyway for short snippets). Please update and explain the why. Thanks!